### PR TITLE
[ISSUE-443][FEATURE] Support customized working dir path

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/StorageManager.scala [hunk 1]
```diff
@@ -289,7 +289,7 @@ private[worker] final class StorageManager(
   val (deviceInfos, diskInfos) = {
     val workingDirInfos =
       RssConf.workerBaseDirs(conf).map { case (workdir, maxSpace, flusherThread, storageType) =>
-        (new File(workdir, RssConf.WorkingDirName), maxSpace, flusherThread, storageType)
```
Comment (waitinfuture) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/StorageManager.scala:292]: We can delete WorkingDirName then

File: common/src/main/scala/com/aliyun/emr/rss/common/RssConf.scala [hunk 2]
```diff
@@ -528,6 +528,10 @@ object RssConf extends Logging {
     conf.getTimeAsMs("rss.expire.nonEmptyDir.duration", "1d")
   }
 
+  def localWorkingDirName(conf: RssConf): String = {
```
Comment (waitinfuture) [inline common/src/main/scala/com/aliyun/emr/rss/common/RssConf.scala:531]: Better to rename to workingDirName, and HDFS/OSS should also respect this config @FMX

Comment (AngersZhuuuu) [reply common/src/main/scala/com/aliyun/emr/rss/common/RssConf.scala]: > workingDirName

Yea， how about current

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**